plugin/main.cpp

```diff
@@ -19,7 +19,8 @@
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QLoggingCategory::setFilterRules(QStringLiteral("geoclue.provider.mlsdb.debug=false\n"
-                                                    "geoclue.provider.mlsdb.position.debug=false"));
+                                                    "geoclue.provider.mlsdb.position.debug=false\n"
+                                                    "geoclue.provider.mlsdb.online.debug=false"));
```

Comment:
I tested that you can turn them on manually via env, and you can.  E.g., doing:

QT_LOGGING_DEBUG=1 QT_LOGGING_RULES="geoclue.provider.mlsdb.online.debug=true" /usr/libexec/geoclue-mlsdb

showed the debug output once more.